Doc/library/tkinter.rst

```diff
@@ -13,12 +13,21 @@
 The :mod:`tkinter` package ("Tk interface") is the standard Python interface to
 the Tk GUI toolkit.  Both Tk and :mod:`tkinter` are available on most Unix
 platforms, as well as on Windows systems.  (Tk itself is not part of Python; it
-is maintained at ActiveState.) You can check that :mod:`tkinter` is properly
-installed on your system by running ``python -m tkinter`` from the command line;
-this should open a window demonstrating a simple Tk interface.
+is maintained at ActiveState.)
+
+Running ``python -m tkinter`` from the command line should open a window
+demonstrating a simple Tk interface, letting you know that :mod:`tkinter` is
+properly installed on your system, and also showing what version of Tcl/Tk is
+installed, so you can read the Tcl/Tk documentation specific to that version.
 
 .. seealso::
 
+   `Tk commands <https://www.tcl.tk/man/tcl8.6/TkCmd/contents.htm>`_
+      Commands without underscores in their name correspond to :mod:`tkinter` or :mod:`tkinter.ttk` widget classes.
```

Comment:
The relation between Tk commands and Tkinter is more complex. Some commands with and without underscores correspond methods in all Tkinter widget classes: `place`, `tk_bisque`. Some commands with underscores correspond classes in submodules: `tk_dialog`, `tk_getOpenFile`. Some method correspond Tk variables, not commands: `tk_strictMotif`.

It would be better to have separate links from the descriptions of every Tkinter class.

Reply:
That sentence could be removed.  Or improved.  "Many commands ... " or "Each tkinter widget class implements one of these commands."

Links could be added for ttk widgets, but the tk widgets are not (currently) documented, that I can find.